docs/tutorials/index.rst

```diff
@@ -0,0 +1,49 @@
+GeoMesa |release| Tutorials
+===========================
+
```

Comment:
Do the tutorials release the ext project to be public, documented, and good-to-go?